cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -200,6 +201,11 @@ public void init() {
                     logger.atInfo().withCause(e).log(re.toString(), e);
                     ctx.status(HttpServletResponse.SC_BAD_REQUEST).json(re);
                 })
+                .exception(InvalidItemException.class, (e, ctx) -> {
+                    RadarError re = new RadarError("Not Found.");
```

Comment:
I'm with Ryan on this. This looks semantically odd. You're asserting the text "not found" but not returning the "NOT_FOUND"error code.

But given the other discussion it would make sense to clarify that the "source" level was not found.

Reply:
Oh this was another copy/paste error. Switched to Bad Request. Invalid Item could also work, but that should already be in the oracle error message.